Makefile

```diff
@@ -178,8 +179,10 @@ $(RELEASE_DIR):
 build-release-artifacts: build-cluster-templates build-infra-yaml build-metadata-yaml build-host-agent-binary
 
 build-cluster-templates: $(RELEASE_DIR) cluster-templates
-	cp $(BYOH_TEMPLATES)/v1beta1/cluster-template.yaml $(RELEASE_DIR)/cluster-template.yaml
-	sed -i -e 1,20d $(RELEASE_DIR)/cluster-template.yaml
+	cp $(BYOH_TEMPLATES)/v1beta1/templates/docker/cluster-template.yaml $(RELEASE_DIR)/cluster-template-docker.yaml
+	sed -i -e 1,20d $(RELEASE_DIR)/cluster-template-docker.yaml
```

Comment:
We paired, and debated that we can avoid doing any `sed` commands by having an additional folder, to separate out the `e2e` and `docker` use cases.

Reply:
This is there to remove resources `ClusterResourceSet` and associated `ConfigMap` in the generated templates, which are not currently being used.

Reply:
We could use `kustomize edit remove resource`. But we think it is better to have separate sources. Created e2e directory for e2e tests - this will not be published during release. The release artifacts are from `docker` and `vm` directories **only**